cocos/base/ccUTF8.h

```diff
@@ -207,13 +207,14 @@ class CC_DLL StringUTF8
     std::size_t length() const;
     void replace(const std::string& newStr);
 
-    std::string getAsCharSequence() const;
+    std::string getAsCharSequence(std::size_t pos = 0, std::size_t len = (std::size_t)-1) const;
```

Comment:
Please, use C++ coding style for cocos2d-x
https://github.com/cocos2d/cocos2d-x/blob/v3/docs/CODING_STYLE.md#default-arguments
Not use default arguments, add new fuction.

Reply:
OK